### ISSUE TITLE
Formatting issue with complex vertical merging

### ISSUE THREAD

**Issue by SimonAkers** [NicoNekoru/obsidan-advanced-table-xt#36]
I have a need for a very specific table, which logically seems like it should work but in practice does not yield the result I would expect.

The table I want is shaped like this:
![Screenshot 2023-11-28 171944](https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/31214887/93954885-dd48-411d-bb82-459832bca59d)
Notice how the cells in columns 4 and 5 are merged, compared to columns 2 and 3.

Logically, we would make this shape with 4 rows and then vertically merge the necessary cells. Doing so in Obsidian with this plugin looks like this:
```
| Col 1 | Col 2 | Col 3 | Col 4 | Col 5 | Col 6 |
| ----- | ----- | ----- | ----- | ----- | ----- |
|       |       |       |       |       |       |
| ^     | ^     | ^     |       |       | ^     |
|       | ^     | ^     | ^     | ^     |       |
| ^     | ^     | ^     |       |       | ^     |
```
![Screenshot 2023-11-28 172232](https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/31214887/1cdd0af4-6ada-48d8-b6b8-ea2326a7495e)

Right away, we can tell that the table looks different from what I'm after. If I begin filling in cells that I need, it starts to look correct:
```
| Col 1 | Col 2 | Col 3 | Col 4 | Col 5 | Col 6 |
| ----- | ----- | ----- | ----- | ----- | ----- |
| X     | X     | X     |       |       | X     |
| ^     | ^     | ^     | X     | X     | ^     |
| X     | ^     | ^     | ^     | ^     | X     |
| ^     | ^     | ^     |       |       | ^     |
```
![Screenshot 2023-11-28 172457](https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/31214887/71839d32-6d46-42d5-be1c-fe8c185e8aac)

But as soon as I fill certain cells, such as in column 4 or 5 on row 1, it breaks down again:
```
| Col 1 | Col 2 | Col 3 | Col 4 | Col 5 | Col 6 |
| ----- | ----- | ----- | ----- | ----- | ----- |
| X     | X     | X     |       | X     | X     |
| ^     | ^     | ^     | X     | X     | ^     |
| X     | ^     | ^     | ^     | ^     | X     |
| ^     | ^     | ^     |       |       | ^     |
```
![Screenshot 2023-11-28 172700](https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/31214887/a94789be-7ad6-4686-8abf-c41e64f47984)

I suspect this has to do with how the cells are anchored/stretched, but I do not know if this is an Obsidian quirk or the plugin itself. I am not sure what can be done about it, or if the behavior is simply expected and what I am doing should fail. A workaround might be to use HTML to create nested tables where needed, but I would prefer to keep my documents without HTML.

Thanks for any help or suggestions!

**Comment by NicoNekoru** [comment]
From what I suspect this is more of a css problem than a problem with this plugin itself. I will try to come up with a fix soon

**Comment by SimonAkers** [comment]
I appreciate it! No rush at all. Your plugin is great!

**Comment by NicoNekoru** [comment]
Should be fixed in 1.2.5 !
